### PR TITLE
HZN-1490 Add real-time valuation notebook

### PR DESCRIPTION
**Pull request by steco**
# Pull Request Checklist

- [x] Changes follow the [style guide](https://github.com/finbourne/sample-notebooks/blob/master/docs/FINBOURNE%20notebook%20style%20guide.ipynb)
- [x] Tests pass
- [x] Raised the PR against the `develop` branch
- [x] Notebook outputs do not contain any priviledged data

# Description of the PR

Added a notebook that demonstrates how to run a real-time valuation in LUSID.

### CODE REVIEW

**Comment by steco** [comment]
1. Should stop the notebook from processing he remaining cells if the environment doesn't have the required identifiers
2. Should populate all the instrument identifiers

File: examples/use-cases/valuation/Real-time Valuation.ipynb [hunk 1]
```diff
@@ -0,0 +1,1491 @@
+{
```
Comment (MikeMcGarry) [inline examples/use-cases/valuation/Real-time Valuation.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #21.        format_quotes_response,
</pre><p>Add format_holdings_response in the import</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/775/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (MikeMcGarry) [inline examples/use-cases/valuation/Real-time Valuation.ipynb:1]: <p>Reach out to <a href="mailto:support@lusid.com" target="_blank">support@lusid.com</a> to configure additional unique identifiers.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/775/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (MikeMcGarry) [inline examples/use-cases/valuation/Real-time Valuation.ipynb:1]: <p>This seems to go to <a href="https://www.lusid.com/app/home" target="_blank">https://www.lusid.com/app/home</a>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/775/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Comment (MikeMcGarry) [inline examples/use-cases/valuation/Real-time Valuation.ipynb:1]: <p>You are loading holdings not transactions.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/775/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Comment (MikeMcGarry) [inline examples/use-cases/valuation/Real-time Valuation.ipynb:1]: <p>Remove output as contains valor number</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/775/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Comment (MikeMcGarry) [inline examples/use-cases/valuation/Real-time Valuation.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #1.    quotes_supplier = "Lusid"
</pre><p>Add some commentary on why you are loading prices when you are streaming them</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/775/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Comment (MikeMcGarry) [inline examples/use-cases/valuation/Real-time Valuation.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #2.    rates = [['GBP/USD', 1.26], ['USD/CHF', 0.87]]
</pre><p>Does SIX not do live exchange rates?</p><p> </p><p>Also need more rates depending on holdings.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/775/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Comment (MikeMcGarry) [inline examples/use-cases/valuation/Real-time Valuation.ipynb:1]: <p>Why does the bond price with no valuation? Makes me think that SIX can't price futures</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/775/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Comment (Seamus-Salt) [reply examples/use-cases/valuation/Real-time Valuation.ipynb:1]: <p>If this is the same holdings as in the pms-demo scope then we also need GBP/CHF, or the recipe needs to be changed to infer missing fx rates.</p>

Comment (steco) [reply examples/use-cases/valuation/Real-time Valuation.ipynb:1]: Yes, as a fall-back in case the market isn't open

Comment (steco) [reply examples/use-cases/valuation/Real-time Valuation.ipynb:1]: Bond future prices for me . . .

**Review by Seamus-Salt — APPROVED** [review]
/LGTM

**Review by AndrewNeudegg — APPROVED** [review]

**Comment by Seamus-Salt** [comment]
Looks good - reviewed but need someone else to approve for merge.

**Pull request merged**